examples/openvino/audio-classification/README.md

```diff
@@ -0,0 +1,103 @@
+<!---
+Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Audio classification examples
+
+This folder contains [`run_audio_classification.py`](https://github.com/huggingface/optimum/blob/main/examples/openvino/audio-classification/run_audio_classification.py), a script to fine-tune a 🤗 Transformers model on the 🗣️ [Keyword Spotting subset](https://huggingface.co/datasets/superb#ks) of the SUPERB dataset while applying quantization aware training (QAT). QAT can be easily applied by replacing the Transformers [`Trainer`](https://huggingface.co/docs/transformers/main/en/main_classes/trainer#trainer) with the Optimum [`OVTrainer`]. Any model from our [hub](https://huggingface.co/models) can be fine-tuned and quantized, as long as the model is supported by the [`AutoModelForAudioClassification`](https://huggingface.co/docs/transformers/main/en/model_doc/auto#transformers.AutoModelForAudioClassification) API.
```

Comment:
>>Any model from our [hub](https://huggingface.co/models) can be fine-tuned and quantized, as long as the model is supported by the [`AutoModelForAudioClassification`]
Is this the correct statement really?

Reply:
`quantization aware training` -> `Quantization-Aware Training`

Reply:
> > > Any model from our [hub](https://huggingface.co/models) can be fine-tuned and quantized, as long as the model is supported by the [`AutoModelForAudioClassification`]
> > > Is this the correct statement really?

My answer is yes. `AutoModelForAudioClassification` API is from `transformers` repo, and always yields a normal PyTorch model. When it is fed into `OVTrainer`, it will be wrapped by NNCF to do quantization and finetuning. We follow the statement from existing openvino examples, e.g., in text-classification.

Reply:
> > > > Any model from our [hub](https://huggingface.co/models) can be fine-tuned and quantized, as long as the model is supported by the [`AutoModelForAudioClassification`]
> > > > Is this the correct statement really?
> 
> My answer is yes. `AutoModelForAudioClassification` API is from `transformers` repo, and always yields a normal PyTorch model. When it is fed into `OVTrainer`, it will be wrapped by NNCF to do quantization and finetuning. We follow the statement from existing openvino examples, e.g., in text-classification.

[Here](https://github.com/huggingface/optimum/blob/v1.7.1/tests/onnxruntime/test_modeling.py#L2368-#L2381) is the list of the architectures that should be supported (architectures for which the ONNX export is validated). The exact list of the architectures supported by `optimum.exporters` can be found [here](https://github.com/huggingface/optimum/blob/1.7.1/optimum/exporters/tasks.py)